### DISCUSSION TITLE
Update documentation for default of updateLockFiles

### DISCUSSION THREAD

**Discussion by L-McHugh** [renovatebot/renovate#26869] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

The documentation for `updateLockFiles` says the default is `True` but when inspecting this further it is [not being set](https://docs.renovatebot.com/renovate-schema.json). I've taken some screenshots of this. 

**The docs:**
![Screenshot 2024-01-26 at 11 37 12](https://github.com/renovatebot/renovate/assets/105207906/bf412da6-51a6-4f76-b0b8-bd5952d18a2d)

**Json**
![Screenshot 2024-01-26 at 11 45 16](https://github.com/renovatebot/renovate/assets/105207906/d6a281de-bd00-42a0-afb3-e50c5f7fe2df)
5207906/7a1fff75-851f-44da-841b-bcc058093a3b)

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
The documentation is correct. Boolean config options without a default specified will default to `true`
